tests/e2e/tutorials/ti-plan.js

```diff
@@ -89,7 +89,7 @@ async function runTutorial() {
     // Click "Load Analysis" button
     const buttonsLoadAnalysis = await utils.getButtonsWithText(postProIframe, "Load Analysis");
     await buttonsLoadAnalysis[0].click();
-    await tutorial.waitFor(20000, "Loading anaylsis");
+    await tutorial.waitFor(20000, "Loading analysis");
```

Comment:
arent' you going to remove the puppeteer scripts?

Reply:
I would keep them for a while... #nostalgic